src/popup/router/index.ts

```diff
@@ -51,7 +51,7 @@ const unbind = router.beforeEach(async (to, from, next) => {
     (
       !RUNNING_IN_POPUP
       && to.name === ROUTE_INDEX
-      && (await browser?.storage.local.get(lastRouteKey))[lastRouteKey]
+      && ((await browser?.storage.local.get(lastRouteKey)) as any)[lastRouteKey]
```

Comment:
aren't we able to define the type here?

Reply:
Not that easy. The typing of the storage getter defines the return value to be `string | number | true | RegExp | ArrayBuffer | StorageObject | DataView | StorageArray | StorageMap | StorageSet | undefined` and the TS engine is complaining that it's not assignable to `NavigationGuardNextCallback`. I hate this `as any` situations, but after spending some time with this I didn't come up with anything meaningful :/